router/router.go

```diff
@@ -100,6 +101,11 @@ func Load() http.Handler {
 	if sunlight {
 		e.GET("/api/repos", api.GetAllRepos)
 		e.GET("/version", web.Version)
+		e.GET("/debug/pprof/", gin.WrapF(pprof.Index))
+		e.GET("/debug/pprof/cmdline", gin.WrapF(pprof.Cmdline))
+		e.GET("/debug/pprof/profile", gin.WrapF(pprof.Profile))
+		e.GET("/debug/pprof/symbol", gin.WrapF(pprof.Symbol))
+		e.GET("/debug/pprof/trace", gin.WrapF(pprof.Trace))
```

Comment:
There are a few more endpoints to include:

```
e.GET("/debug/pprof/heap", gin.WrapF(pprof.Handler("heap"))))
e.GET("/debug/pprof/goroutine", gin.WrapF(pprof.Handler("goroutine")))
e.GET("/debug/pprof/threadcreate", gin.WrapF(pprof.Handler("threadcreate")))
e.GET("/debug/pprof/block", gin.WrapF(pprof.Handler("block")))
```

Reply:
There's also mutex: 

```
e.GET("/debug/pprof/mutex", gin.WrapF(pprof.Handler("mutex")))
```

Reply:
Building the list of pprof.Handler endpoints dynamically:

first add a new import:

```
rpprof "runtime/pprof"
```

```
for p := range rpprof.Profiles() {
    e.GET("/debug/pprof/"+p.Name(),gin.WrapF(pprof.Handler(p.Name())))
}
```